functionnal_tests/cypress/integration/tags/tags.spec.js

```diff
@@ -0,0 +1,113 @@
+import { PAGES } from '../../support/urls_commands'
+
+// TODO
+// * Check create new tag
+//  ** Create a new tag
+//  ** click on "validate"
+//  ** check if new tag has been created
+
+// * Unchecking a tag
+//  ** click on a checked item on the list
+//  ** check if it is still marked as a content tag
+
+// * Re-checking a tag
+//  ** go back to the unchecked tag
+//  ** check it
+//  **  see if it is marked as a content tag
```

Comment:
Forgot comments?

Reply:
yes, these tests are being fixed so there will be a new version soon

Reply:
I am leaving this here, this is very temporary.